native/package.json

```
@@ -7,26 +7,38 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
-    "test": "jest"
-
+    "test": "jest",
+    "tsc": "tsc",
+    "lint": "eslint .",
```

Should we add prettier here too?
```suggestion
    "lint": "eslint . && prettier -c .",
```

---

Good suggestions, thanks!